test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -1,5 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+Import-Module (Join-Path -Path $PSScriptRoot '..\Microsoft.PowerShell.Security\certificateCommon.psm1') -Force
+Import-Module (Join-Path -Path $PSScriptRoot '..\..\..\Tools\Modules\HelpersCommon\HelpersCommon.psm1') -Force
```

Comment:
HelperCommon is in the path, and don't force, we care about perf in our tests.
```suggestion
Import-Module HelpersCommon
```

Reply:
addressed